### PR TITLE
Update tutorial-co-manage-clients.md

### PR DESCRIPTION
**Pull request by byteben**
You don’t need to configure the MDM User Scope when ConfigMgr enrolls the client - the URLs are obtained from the ConfigMgr DB

### CODE REVIEW

**Comment by PRMerger17** [comment]
@byteben : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@aczechowski : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by aczechowski — APPROVED** [review]
Thanks @byteben for the contribution (and apologies for the delay). I made a few edits for style.

**Comment by aczechowski** [comment]
#sign-off on behalf of @mestew (discussed with her on the phone)

**Comment by PRMerger14** [comment]
@aczechowski: I'm sorry - only the author of this article, @mestew, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by mestew** [comment]
#sign-off

File: memdocs/configmgr/comanage/tutorial-co-manage-clients.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ Automatic enrollment also lets users enroll their Windows 10 devices to Intune.
 
    > [!IMPORTANT]  
    > If both **MAM user scope** and automatic MDM enrollment (**MDM user scope**) are enabled for a group, only MAM is enabled. Only Mobile Application Management (MAM) is added for users in that group when they workplace join personal device. Devices aren't automatically MDM-enrolled.  
+   >
+   > When Configuration Manager is set to enroll devices to Intune, ou don't need to change the MDM user scope for device token enrollment. Configuration Manager uses the MDM URLs that it stores in the site database.
```
Comment (ShannonLeavitt) [inline memdocs/configmgr/comanage/tutorial-co-manage-clients.md:146]:
```suggestion
   > When Configuration Manager is set to enroll devices to Intune, you don't need to change the MDM user scope for device token enrollment. Configuration Manager uses the MDM URLs that it stores in the site database.
```

**Comment by PRMerger18** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**